### DISCUSSION TITLE
Course about Git & GitHub

### DISCUSSION THREAD

**Discussion by Ptr-Parker** [community/community#133932] [New to GitHub] [answered]
Can anyone recommend me a course about Git & GitHub?

**Comment by mrptrprkr** [comment, marked as answer]
Here is a Udemy Course, I would recommend: https://www.udemy.com/course/git-and-github-bootcamp/

**Comment by ghostinhershell** [comment]
Hi @Ptr-Parker, 

Thanks so much for posting in our community. Getting GitHub certified is a resounding endorsement that validates your skills, credibility, trust, and knowledge of the technologies and developer tools that are used by more than 100 million developers worldwide. You can learn more here: https://resources.github.com/learn/certifications/.
